handlers/proxy_handler.cc

```diff
@@ -0,0 +1,240 @@
+#include "proxy_handler.h"
+
+#include "../http/httpRequest.h"
+#include "../http/httpResponse.h"
+#include "../http/http.h"
+
+#include <boost/array.hpp>
+#include <boost/asio.hpp>
+#include <boost/algorithm/string/trim.hpp>
+using boost::asio::ip::tcp;
+
+RequestHandler::Status ProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+	this->m_uri_prefix_ = uri_prefix;
```

Comment:
Yes! haha I started doing this before you merged your whitespace fix, I only did tabs because I though you guys did tabs.